app/assets/javascripts/admin/addon/components/admin-flag-item.gjs

```diff
@@ -96,20 +96,26 @@ export default class AdminFlagItem extends Component {
   @action
   delete() {
     this.isSaved = false;
-    this.dialog.yesNoConfirm({
+    return this.dialog.yesNoConfirm({
```

Comment:
I always get comments now to convert AJAX calls to async/await, probably worth doing this here